### ISSUE TITLE
Failed to create dataset

### ISSUE THREAD

**Issue by Chamais** [firebase/mlkit-custom-image-classifier#11]
Hello, I am not an expert in flutter but I am very interested in this application and I have been trying to test it without results for more than a week. I have followed all the indications of the Getting Started and reviewed them several times but I get nothing but the "View Tutorial" screen and the "Add a new dataset" screen. I fill in the Title and Description data and receive the following message: Exception: Failed to create dataset. <! DOCTYPE html>
I / flutter (11395): <html lang = en>
I / flutter (11395): <meta charset = utf-8>
I / flutter (11395): <meta name = viewport content = "initial-scale = 1, minimum-scale = 1, width = device-width">
I / flutter (11395): <title> Error 404 (Page not found) !! 1 </title> .............
I also receive these messages in the terminal: W / Firestore (11395): (18.2.0) [Firestore]: Listen for Query (datasets order by __name__) failed: Status {code = PERMISSION_DENIED, description = Missing or insufficient permissions., Cause = null}
I / System.out (11395): com.google.firebase.firestore.FirebaseFirestoreException: PERMISSION_DENIED: Missing or insufficient permissions.
I have a Blaze Plan and I have followed the steps to authorize access, but I'm sure I'm doing something wrong and I don't know what it can be. Could you help me??

**Comment by prakhar1989** [comment]
Are you signed in to the app? Also, did you try running `npm run deploy` and did that succeed?

**Comment by Chamais** [comment]
Hi Prakhar, I followed all the steps indicated in your article except for the processes to send emails via my Firebase functions. After analyzing several times that I could do wrong, I chose to incorporate the processes for email management and everything worked perfectly. Even after incorporating Ganesh improvements to facilitate loading images. Everything works perfectly! Thank you very much for this great work that gives us all great peace of mind and functionality.
Congratulations and greetings!
